mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICalendarParser.java

```diff
@@ -72,15 +73,6 @@
     public static final String SOURCE_ATTRIBUTE_PARAMETER_DEFAULT_VALUE = "icsAttachments";
     public static final String DESTINATION_ATTRIBUTE_PARAMETER_DEFAULT_VALUE = "calendars";
 
-    static {
-        CompatibilityHints.setHintEnabled(CompatibilityHints.KEY_RELAXED_PARSING, true);
-        CompatibilityHints.setHintEnabled(CompatibilityHints.KEY_RELAXED_UNFOLDING, true);
-        CompatibilityHints.setHintEnabled(CompatibilityHints.KEY_RELAXED_VALIDATION, true);
-        CompatibilityHints.setHintEnabled(CompatibilityHints.KEY_OUTLOOK_COMPATIBILITY, true);
-        CompatibilityHints.setHintEnabled(CompatibilityHints.KEY_NOTES_COMPATIBILITY, true);
-        CompatibilityHints.setHintEnabled(CompatibilityHints.KEY_VCARD_COMPATIBILITY, true);
-    }
```

Comment:
why not using this: https://github.com/ical4j/ical4j/issues/195#issuecomment-393916985 ?

Reply:
System properties seems uggly to me. Furthermore, which class should I set this in? All class in  that package depends on that property, and I want to ensure DEV environment is the same than prod, whatever ICAL4J related class you are running.

Properties file was just better at this, hence my choice.

Reply:
Did I read "properties file" and "better" in the same sentence ?
Ok, so now we can't define a ical4j property without loosing the entire default configuration.
A user can break James just by putting a file without the cache.impl entry on the classpath, etc.

So I would rather reconsider this choice. WDYT ?

Reply:
Yes I would also prefer to see this setProperty call in this class (ICalendarParser) than in a properties file.